src/XMLimport.cpp

```diff
@@ -1311,11 +1337,8 @@ void XMLimport::readTriggerGroup( TTrigger * pParent )
                 pT->setName( readElementText() );
                 continue;
             }
-
-            else if( name() == "script")
-            {
-                pT->mScript = readElementText();
-                continue;
+            else if( name() == "script") {
+                tempScript = readElementText();
```

Comment:
The previous code set the data read here directly into the Trigger's script area - bypassing the compilation step that would report an error if it didn't compile that would occur should the "proper" (?) setter `pT->setScript(...)` have been used. Now we save it and use the "right" way at the end of parsing the trigger.